src/state/actions/dispatchArNSUpdate.ts

```diff
@@ -40,6 +44,37 @@ export function dispatchArNSUpdate({
       payload: totalIds,
     });
   });
+  const errorHandler = (e: string) => {
+    if (e.startsWith('Error getting ArNS records')) {
+      eventEmitter.emit(
+        'error',
+        new Error('Unable to load ArNS records. Please refresh to try again.'),
+      );
+      // Behaviour note: this will only stop the loading UI, it will still display the
```

Comment:
This comment just ends with "the", seems like some more is meant to be mentioned?

Reply:
Realized the comment was incorrect anyways, removing.